src/Messages.css

```diff
@@ -0,0 +1,75 @@
+
+.PageContainer{
+	font-size: 15px;
+	font-weight: 600;
+	padding: 2%;
+	width:50%;
+	height:70%;
+	margin-left:auto;
+	margin-right:auto;
+	align-items: center;
+}
+
+.MessageContainer{
+	background-color: #f3f3f3;
+	height:500px;
+	margin-top:5%;
+	margin-bottom:10%;
+	margin-left:auto;
+	margin-right:auto;
+	overflow: auto;
+	font-weight: 600;
+	border-radius: 10px;
+}
+
+.UserOne {
+	background-color: #F8DABB;
+	padding: 15px;
+	width: 70%;
+	margin: 10px;
+	border-radius: 8px;
+	float:right;
+	color: #303030;
+	font-size: 14px;
+}
+.UserTwo {
+	background-color: #4DADB7;
+	padding: 15px;
+	width: 70%;
+	margin: 10px;
+	border-radius: 8px;
+	float:left;
+	color: #303030;
+	align-content: right;
+	font-size: 14px;
+}
+
+
+.MessageContainer input[name="message"] {
+	bottom: 5px;
+	width: 70%;
+	padding: 5px;
+	margin-left: 5px;
+	border-radius: 5px;
+	position: sticky;
+	border-color:#0081a7;
+}
+.MessageContainer button{
+    color: white;;
```

Comment:
Maybe use a different color for the time text? It isn't as clear in the coral message box so it's harder to see

Reply:
Can the button align to the right of the conversation container so they're better spaced out?